initramfs/init.sh

```diff
@@ -21,7 +21,7 @@ rwmode=""
 
 shutdown()
 {
-	while [ 1 ]; do
+	while : ; do
```

Comment:
: is reasonably supported, but not fully compliant, also hard to read imo.
besides, shellcheck said to just use true!
```sh
	while [ 1 ] ; do
                ^-- SC2161: Instead of '[ 1 ]', use 'true'.
```
```sh
while true; do
done
```
is fine no?